nixos/modules/installer/tools/nixos-install.sh

```diff
@@ -92,13 +92,14 @@ fi
 mkdir -m 0755 -p $mountPoint/dev $mountPoint/proc $mountPoint/sys $mountPoint/etc $mountPoint/run $mountPoint/home
 mkdir -m 01777 -p $mountPoint/tmp
 mkdir -m 0755 -p $mountPoint/tmp/root
-mkdir -m 0755 -p $mountPoint/var
+mkdir -m 0755 -p $mountPoint/var/permissions-wrappers
```

Comment:
why do we need this mkdir now if we didn't need this before for `/var/setuid-wrappers`?